Detectors/TRD/simulation/include/TRDSimulation/Detector.h

```diff
@@ -93,6 +93,9 @@ class Detector : public o2::Base::DetImpl<Detector>
   float mGasNobleFraction;
   float mGasDensity;
 
+  bool mTRon; //  Switch for TR simulation
+  int mWion;  //  Ionization potential
```

Comment:
probably `int` is wrong since you assign floating point numbers later on??

Reply:
Not probably, it is plain wrong. I am fixing this and adding a bit of formatting.

Reply:
Thanks